.stylelintrc.json

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": ["stylelint-config-standard", "stylelint-config-prettier"]
+  "extends": ["stylelint-config-standard", "stylelint-config-prettier"],
+  "rules": {
+    "selector-class-pattern": [
+      "^[a-z]([-]?[a-z0-9]+)*(__[a-z0-9]([-]?[a-z0-9]+)*)?(--[a-z0-9]([-|_]?[a-z0-9]+)*)?$",
+      {
+        "resolveNestedSelectors": true
+      }
```

Comment:
I would add over here:
```
"message": "Expected class selector to be in kebab-case or BEM-style naming."
```
To prevent an output like this:
```
Expected ".-not-this" to match pattern "^[a-z]([-]?[a-z0-9]+)*(__[a-z0-9]([-]?[a-z0-9]+)*)?(--[a-z0-9]([-|_]?[a-z0-9]+)*)?$"
```